assembly/src/parsers/u32_ops.rs

```diff
@@ -566,9 +569,9 @@ pub fn parse_u32rotl(span_ops: &mut Vec<Operation>, op: &Token) -> Result<(), As
 /// 0-31 and the value to be shifted is asserted to be a 32-bit value.
 ///
 /// VM cycles per mode:
-/// - u32rotr: 16 cycles
-/// - u32rotr.b: 6 cycles
-/// - u32rotr.unsafe: 7 cycles
+/// - u32rotr: 58 cycles
+/// - u32rotr.b: 48 cycles
+/// - u32rotr.unsafe: 44 cycles
```

Comment:
Same comment regarding `u32rotr.b` cycle count

Reply:
I missed it. Thanks. Fixed.